README.md

```diff
@@ -22,3 +22,155 @@ e.g.
 ```
 npm i @fastify/cors -w @mykomap/back-end --save
 ```
+
+## Installation and updates
+
+Conceptually, installation of these applications require:
```

Comment:
I think including some links to sections of the doc at the start would make it more readable and easier to navigate. The sections I can think of are:

- Overview
- How to install
  - Requirements
  - Env variables
  - Installation steps
- How to update